lib/pbench/agent/tool_meister.py

```diff
@@ -422,39 +423,49 @@ def __init__(self, name, tool_opts, logger=None, **kwargs):
         self.args = None
         self.process = None
 
+    def log_subprocess_output(self, pipe):
+        for line in iter(pipe.readline, b""):
```

Comment:
Just for my own edification, why is this idiom better than just
```
        for line in pipe.readline:
```
?

Reply:
hmm I dont know, `iter` is just something that occurred to me first I guess. But yeah I think `readlines()` is a simpler implementation to read lines.